content/doc/book/managing/plugins.adoc

```diff
@@ -172,6 +172,27 @@ By default, the Jenkins controller will check for updates from the Update Center
 once every 24 hours. To manually trigger a check for updates, simply click on
 the *Check now* button in the *Updates* tab.
 
+=== List installed plugins and their versions
+
+The simplest way to retrieve the list of installed plugins and their versions is by using the Jenkins Script Console.
+
+Follow these steps:
+
+1. Open the Jenkins link:/doc/book/managing/script-console/[Script Console]:
+
+   - Navigate to `Manage Jenkins` > `Script Console`.
+2. Run the following script to list installed plugins and their versions:
+
+
+   [source,groovy]
```

Comment:
Leading spaces cause it to render incorrectly.  Visible on the [prototype site](https://deploy-preview-6973--jenkins-io-site-pr.netlify.app/doc/book/managing/plugins/#list-installed-plugins-and-their-versions).
```suggestion
[source,groovy]
```